its/src/test/java/com/sonar/it/csharp/MultipleProjectsTest.java

```diff
@@ -0,0 +1,424 @@
+
+  private static final String FIRST_PROJECT_DIRECTORY = "MultipleProjects:FirstProject";
+  private static final String FIRST_PROJECT_FIRST_CLASS_FILE = "MultipleProjects:FirstProject/FirstClass.cs";
+  private static final String FIRST_PROJECT_SECOND_CLASS_FILE = "MultipleProjects:FirstProject/SecondClass.cs";
+  private static final String FIRST_PROJECT_ASSEMBLY_INFO_FILE = "MultipleProjects:FirstProject/Properties/AssemblyInfo.cs";
+
+  private static final String SECOND_PROJECT_DIRECTORY = "MultipleProjects:SecondProject";
+  private static final String SECOND_PROJECT_FIRST_CLASS_FILE = "MultipleProjects:SecondProject/FirstClass.cs";
+
+  @ClassRule
+  public static RuleChain chain = getRuleChain();
+
+  private static RuleChain getRuleChain() {
+    assertThat(SystemUtils.IS_OS_WINDOWS).withFailMessage("OS should be Windows.").isTrue();
+
+    // Scanner for MSBuild caches the analyzer, so running the test twice in a row means the old binary is used.
+    // This code deletes the cache, but there should be a way to run without cache.
+    // Ticket: https://jira.sonarsource.com/browse/SONARMSBRU-346
+    String localAppData = System.getenv("LOCALAPPDATA") + "\\Temp\\.sonarqube";
+    try {
+      FileUtils.deleteDirectory(new File(localAppData));
+    }
+    catch (IOException ioe) {
+      throw new IllegalStateException("could not delete Scanner for MSBuild cache folder", ioe);
+    }
+
+    return RuleChain
+      .outerRule(ORCHESTRATOR)
+      .around(temp)
+      .around(new ExternalResource() {
+        @Override
+        protected void before() throws Throwable {
+          ORCHESTRATOR.resetData();
+
+          Path projectDir = Tests.projectDir(temp, PROJECT);
+
+          ScannerForMSBuild beginStep = TestUtils.createBeginStep(PROJECT, projectDir);
+
+          CommandExecutor.create().execute(Command.create("nuget")
+            .addArguments("restore")
+            .setDirectory(projectDir.toFile()),
+            10 * 60 * 1000);
+
+          ORCHESTRATOR.executeBuild(beginStep);
+
+          TestUtils.runMSBuild(ORCHESTRATOR, projectDir, "/t:Rebuild");
+
+          ORCHESTRATOR.executeBuild(TestUtils.createEndStep(projectDir));
+        }
+      });
+  }
+
+  @Test
+  public void projectIsAnalyzed() {
+    assertThat(getComponent(PROJECT).getName()).isEqualTo("MultipleProjects");
+
+    assertThat(getComponent(FIRST_PROJECT_DIRECTORY).getName()).isEqualTo("FirstProject");
+    assertThat(getComponent(FIRST_PROJECT_FIRST_CLASS_FILE).getName()).isEqualTo("FirstClass.cs");
+    assertThat(getComponent(FIRST_PROJECT_SECOND_CLASS_FILE).getName()).isEqualTo("SecondClass.cs");
+    assertThat(getComponent(FIRST_PROJECT_ASSEMBLY_INFO_FILE).getName()).isEqualTo("AssemblyInfo.cs");
+
+    assertThat(getComponent(SECOND_PROJECT_DIRECTORY).getName()).isEqualTo("SecondProject");
+    assertThat(getComponent(SECOND_PROJECT_FIRST_CLASS_FILE).getName()).isEqualTo("FirstClass.cs");
+  }
+
+  @Test
+  public void barIssuesAreFound() {
```

Comment:
sorry, the name of this class and `simpleClassIssueIsFound` are an artefact before a refactoring I will change it